app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -158,6 +158,12 @@ Most teaching vacancies are advertised in March, April or the first half of May,
 
 Set up a Teaching Vacancies job alert to be updated about opportunities throughout the year.
 
+You can search for teaching jobs by region, subject specialism and age group.
```

Comment:
@philspray Something odd is going on here - this PR has a LOT of commits and this content has been duplicated on this page 

![image](https://user-images.githubusercontent.com/87642394/214381932-c30cdc70-5bd1-4ff8-902c-ac02e82e7d84.png)

Firstly, this change needs removing - and then I think it might be worth just flagging with Ross to make sure nothing else weird is going on!